docs/USING_ACTIONS.md

```diff
@@ -0,0 +1,65 @@
+## Actions
+
+Each of the reducers: grid, pager, selection, etc have supporting actions. You may wish to dispatch these actions from other areas of your application outside of the grid. For instance, you could unselect all the rows of the grid that has checkbox column.
+
+
+## Selection Actions
+
+Add the actions
+
+```
+import { Actions } from 'react-redux-grid';
+```
+
+Map them to this.props
+
+```
+const mapDispatchToProps = (dispatch) => {
+  return {
+    deselectAll: stateKey => {
+      dispatch(Actions.SelectionActions.deselectAll(stateKey))
+    },
+  }
+}
+```
+
+Use them in your components from this.props
+
+```
+uncheckAllSelectedRows(event){
```

Comment:
Let's remove everything from `Use them in your components from this.props` down, in this file. Once merged, I'll go back and provide the use case for how best to use the actions.